swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition = 0):
+        super().__init__(category, condition)
+        self.category = "Clothing" 
```

Comment:
Nice work completing your project! You handled the default argument on like 4 so you don't need to set `self.category` to "Clothing" twice like you did on line 6.